### DISCUSSION TITLE
Adding second controller to HA

### DISCUSSION THREAD

**Discussion by PerfectBake420** [vinteo/hass-opensprinkler#179] [Q&A] [answered]
Hello. I am wondering if it is possible to integrate 2 separate OpenSprinklers? I have 1 integrated with the 72 stations it can hold. I need to add another 72 stations. Can this be done?

**Comment by travisghansen** [comment, marked as answer]
Yes there’s nothing preventing that.

**Comment by PerfectBake420** [comment]
Thank you. How do I do it? When I added the integration in, I am sure it
asked me the ip address. How do I add another?

**Comment by travisghansen** [comment]
Do the same thing, put in the IP address of the 2nd device..

**Comment by PerfectBake420** [comment]
Ok. How do I get that input box to come up again? Do I have to delete the I
tegration and add it again?

**Comment by travisghansen** [comment]
Go to the integrations screen and click 'add integration', find opensprinkler, fill out the form. You do not need to remove anything..
